static/scss/main.scss

```diff
@@ -60,6 +60,10 @@ body {
   margin: 0 auto;
 }
 
+.twitter-tweet {
+  margin: 0 auto;
```

Comment:
This seems more fitting to be a utility class, such as `.centralize` or something.

Reply:
That class is added by twitter 😅